pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -83,7 +82,7 @@ func (i *InTotoIte6) generateAttestationFromTaskRun(tr *v1beta1.TaskRun) (interf
 			Builder: slsa.ProvenanceBuilder{
 				ID: i.builderID,
 			},
-			BuildType:   tektonID,
+			BuildType:   fmt.Sprintf("%s/kind=%s", tr.GetGroupVersionKind().GroupVersion().String(), tr.GetGroupVersionKind().Kind),
```

Comment:
Should we just simplify this to `<group>/<version>/<kind>`? Does `kind=` serve a special purpose anywhere else (i.e. parsing, etc)?

Reply:
> Does kind= serve a special purpose anywhere else (i.e. parsing, etc)?

No off the top of my head. I added `kind=` to just try to match what [GetGroupVersionKind().String()](https://github.com/kubernetes/apimachinery/blob/v0.24.3/pkg/runtime/schema/group_version.go#L161-L163) returns but no space there. 

> Should we just simplify this to <group>/<version>/<kind>? 

SGTM! Changed.